src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -861,6 +861,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                             ->end()
                         ->end()
                         ->arrayNode('auto_mapping')
+                            ->info('A collection of "validator.auto_mapper" services per namespace for which auto-mapping must be enabled by default, or null to opt-in with the EnableAutoMapping constraint.')
```

Comment:
You improved upon my wording a lot. Here's one last tweak.

```suggestion
                            ->info('A collection of namespaces for which auto-mapping will be enabled by default, or null to opt-in with the EnableAutoMapping constraint.')
```